docs/tutorials/configure-external-signer-tls.md

```diff
@@ -173,6 +164,5 @@ teku --network=goerli \
 
 [Web3Signer]: https://docs.web3signer.consensys.net/en/latest/
 [Hyperledger Besu]: https://besu.hyperledger.org/development/public-networks/get-started/install
-[Infura]: https://infura.io/
-[ETH1 Goerli node]: https://besu.hyperledger.org/development/public-networks/get-started/start-node#run-a-node-on-goerli-testnet
+[Configure Besu on Holesky]: https://besu.hyperledger.org/development/public-networks/get-started/start-node#run-a-node-on-holesky-testnet
```

Comment:
```suggestion
```

Don't need this link reference at all since it's no longer referenced anywhere on this page.

Reply:
Yeah, I thought maybe we wanted to have it persist somewhere, since we mention Holesky in the doc, but don't link to it anywhere. Should we write a line so that the link is readily provided for a user, or will they just search for Holesky on their own?